backend/src/test/controller/PromotionController.test.ts

```diff
@@ -40,9 +44,16 @@ describe('Unit tests for PromotionController', function () {
     const promotion = new PromotionFactory().generate(user, discount, [
       schedule,
     ]);
```

Comment:
Could we modify the factory method to set the `lat/lon` values for us? Because I noticed that a lot of the tests now have to maintain the responsibility to remember setting lat lon values for a promotion. What do you think about the suggestion?